src/components/TopBar.js

```diff
@@ -33,18 +33,24 @@ export const TopBar = hh(class TopBar extends Component {
         {
           style: {
             display: 'flex', position: 'absolute', left: 0, right: 0, top: 0, bottom: 0,
-            overflow: 'auto'
-          }
+            overflow: 'auto', cursor: 'pointer'
+          },
+          onClick: () => this.hideNav()
         },
         [
           div({
             style: {
-              boxShadow: '3px 0 13px 0 rgba(0,0,0,0.3)', width: 275,
-              backgroundColor: Style.colors.primary,
-              position: 'fixed', height: '100%'
-            }
-          }),
-          div({ style: { width: 275, color: 'white', position: 'absolute' } }, [
+              zIndex: 1,
+              // MP: Rotating the 'angle' icon for breadcrumbs (via shape='angle right' as we're
+              // doing, other rotation options in the icons library, or manually rotating ourselves
+              // via transform: rotate(90deg) on the container) is causing it to appear above this
+              // overlay. It seems to get a tiny z-index value somehow.
```

Comment:
Transforms create a new stacking context just like a `position: absolute`, etc. [[1](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context)] Stacking is ordered, so later elements on the page stack on top of earlier elements. Since `main-menu-container` appears before `root`, this stacks under the transformed icon. I think the z-index is a reasonable workaround in this case, but I might update this comment to say that.